databases/models/TopicInvitations.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+const {Model} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class TopicInvitations extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(_models) {
+      // define association here
+    }
+  }
+  TopicInvitations.init(
+    {
+      topic_invitations_id: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        primaryKey: true
+      },
+      user_id_inviter: {type: DataTypes.STRING, allowNull: false},
+      user_id_invited: {type: DataTypes.STRING, allowNull: false},
+      topic_id: {type: DataTypes.BIGINT, allowNull: false},
+      createdAt: {
+        type: DataTypes.DATE,
+        field: 'created_at',
+        allowNull: false
+      },
+      updatedAt: {
+        type: DataTypes.DATE,
+        field: 'updated_at',
+        allowNull: false
+      }
+    },
+    {
+      sequelize,
+      modelName: 'TopicInvitations',
+      tableName: 'topic_invitations'
+    }
+  );
+  return TopicInvitations;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new Sequelize model `TopicInvitations` seems to be well defined with appropriate data types and constraints. However, the `associate` method is currently empty. If there are any relationships between this model and others (like User or Topic), they should be defined here. If not, you can remove this method to avoid confusion.
```diff
    static associate(_models) {
      // define association here
    }
```
If there are no associations, it would look like this:
```diff
- 10:     static associate(_models) {
- 11:       // define association here
- 12:     }
```
Also, consider adding indexes for fields that will be frequently queried, such as `user_id_inviter`, `user_id_invited`, and `topic_id`. This can significantly improve query performance.


<!-- This is an auto-generated comment by OSS CodeRabbit -->